### PR TITLE
chore: removed Unnecessary Comment

### PR DESCRIPTION
**Pull request by ShashwatPS**
Description :
- Removed the commented out code in library/src/components/layouts/CircleLayout.tsx.

Related issue(s) :
- Resolves Remove this commented out code. #35

### CODE REVIEW

**Review by jonaslagoni — APPROVED** [review]
Changed it to `chore` as we dont need to release anything 🙂 

Nice @ShashwatPS 👍

File: library/src/components/layouts/CircleLayout.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ function calculateLayout(
   elements: Node[],
   passedOptions: Partial<CircleLayoutOptions> = defaultOptions,
 ): FlowElement[] {
-  // const options = { ...defaultOptions, ...passedOptions };
+
```
Comment (jonaslagoni) [inline library/src/components/layouts/CircleLayout.tsx:21]: Think the linter complains that there are an empty line here 🙂

**Review by jonaslagoni — CHANGES_REQUESTED** [review]

File: library/src/components/layouts/CircleLayout.tsx (hunk 1 above)
Comment (ShashwatPS) [reply library/src/components/layouts/CircleLayout.tsx:21]: So i make a PR after fixing it right ?

**Review by jonaslagoni — APPROVED** [review]
👍

**Comment by jonaslagoni** [comment]
@all-contributors please add @ShashwatPS for code

**Pull request merged**